packages/preview-middleware/templates/test/qunit.js

```diff
@@ -11,18 +11,21 @@ sap.ui.loader.config({
         },
         "sap/ui/qunit/sinon-qunit-bridge": {
             deps: ["sap/ui/thirdparty/qunit", "sap/ui/thirdparty/sinon-4"]
```

Comment:
Why is this the only shim using `qunit` instead of `qunit-2`?

Reply:
Oh I think you are right. I'll fix it as part of this PR as well 👍🏻